src/main/resources/static/data-upload.js

```diff
@@ -110,7 +107,7 @@ function uploadFile() {
             },
             error: function (jqXHR, textStatus, errorThrown) {
                 // Handle upload error
-                var message = 'Hmm, the data could not be uploaded. Have you checked the data format, which should be plain, comma delimited or tab delimited. Also do make sure the file size does not exceed 300MB. Let\'s give it another try:) ';
+                var message = 'Data upload failed. Please check the data format, which should be a plain, tab-delimited file. The size of the file should be less than 10MB. ';
```

Comment:
Now you just need to adjust allowed upload size in the properties file - you set it to 300MB, and while you adjusted the message to the desired value, uploads up to 300 MB will still be accepted.